src/System.Private.Xml/src/System/Xml/Serialization/Compiler.cs

```diff
@@ -27,9 +27,6 @@ namespace System.Xml.Serialization
 
     internal class Compiler
     {
-#if !XMLSERIALIZERGENERATOR
-        private bool _debugEnabled = DiagnosticsSwitches.KeepTempFiles.Enabled;
-#endif
```

Comment:
@krwq, @sepidehMS, are there any builds where we don't set XMLSERIALIZERGENERATOR?  I'm wondering if we could remove the define across the codebase.